src/Client.php

```diff
@@ -436,6 +436,10 @@ public function takeScreenshot($saveAs = null)
     {
         $this->start();
 
+        if ($saveAs && ($dir = $_SERVER['PANTHER_SCREENSHOT_DIR'] ?? false)) {
```

Comment:
Shouldn't you test if it's not already an absolute or relative path that is passed?

Reply:
The directory might not exist so I can't check that - are you thinking just to look if the string starts with a `.`?

Reply:
You can use this helper: https://symfony.com/doc/current/components/filesystem.html#isabsolutepath

Reply:
What if someone has the `$_SERVER['PANTHER_SCREENSHOT_DIR']` set and they pass an "absolute" path (ie `takeScreenshot('/screenshot.jpg')`)?

Reply:
Then the screenshot should be stored at `/` as it's an absolute path, isn't it?

Reply:
Yep ok, I currently treat [this as relative](https://github.com/symfony/panther/pull/392/files#diff-ce3bd06cad1893ba2abb840a3061bb39e857377c5feeddfdbdee1bd03b51be81R68). I'll make the change.